sdk/burdock/query/sql/private/query.py

```diff
@@ -52,17 +53,40 @@ def _apply_noise(self, subquery, query, syms, types, sens, srs):
             name, sym = nsym
             name = name.lower()
             sens = sym.sensitivity()
+            
             if sym.type() == "int":
```

Comment:
Does typed recordset coerce these to ints, or should we do it here?  The new release() function doesn't return ints

Reply:
Typed rowset does the coercing in line #99 of rowset.py. So we need not do it here. 
self.m_cols[cn] = np.array(value).astype(int)